src/module_wrap.cc

```diff
@@ -323,12 +323,12 @@ struct file_check {
   bool failed = true;
   uv_file file = -1;
 };
-inline const struct file_check check_file(URL search,
+inline const struct file_check check_file(URL* search,
```

Comment:
I think none of the methods here in question actually modify the incoming parameter? In that case, just switching `URL` → `const URL&` would be nicer…

Reply:
OK. I'll update.